### DISCUSSION TITLE
Lesson 1: ParserError: Expected identifier but got '(' people.push(People(_favoriteNumber, _name)); ^

### DISCUSSION THREAD

**Discussion by LegalTechsis** [smartcontractkit/full-blockchain-solidity-course-py#1324] [Q&A] [answered]
Can someone please assist me with identifying this error? Thank You.

![image](https://user-images.githubusercontent.com/98588810/159467680-aa527b50-2411-4f5f-810d-1aee73f5afc2.png)

**Comment by cromewar** [comment, marked as answer]
Hello @LegalTechsis you have a duplicated sentence on line 7, just delete it.
